### PR TITLE
support kmsg log

### PR DESCRIPTION
**Pull request by bjqian**

### CODE REVIEW

File: pkg/checkers/oom/oom.go [hunk 1]
```diff
@@ -33,9 +34,18 @@ func (c *OOMChecker) Name() string {
 }
 
 func New() *OOMChecker {
-	//todo: support other logpath
+	paths := []string{kmsgLogPath, ubuntuLogPath}
+	for _, path := range paths {
+		file, err := os.Open(path)
+		file.Close()
+		if err != nil {
+			return &OOMChecker{
```
Comment (ArchangelSDY) [inline pkg/checkers/oom/oom.go:42]: Looks confusing. How about we return if it passes the check?

```
if file, err := os.Open(path); err == nil {
    file.Close()
    return &OOMChecker{kernLogPath: path}
}
```

File: pkg/checkers/oom/oom_test.go [hunk 2]
```diff
@@ -26,10 +26,7 @@ func TestCheckOOMLogWhenOOM(t *testing.T) {
 		}
 		check := OOMChecker{kernLogPath: tmp.Name()}
 		defer func() {
-			e := os.Remove(check.kernLogPath)
-			if e != nil {
-				t.Errorf(e.Error())
-			}
+			os.Remove(check.kernLogPath)
```
Comment (ArchangelSDY) [inline pkg/checkers/oom/oom_test.go:29]: Will this remove `/dev/kmsg`?

Comment (bjqian) [reply pkg/checkers/oom/oom_test.go:29]: Nope. It's using the tmp file

File: pkg/checkers/oom/oom.go (hunk 1 above)
Comment (bjqian) [reply pkg/checkers/oom/oom.go:41]: Fixed.

**Review by ArchangelSDY — APPROVED** [review]

**Pull request merged**